### PR TITLE
:art: feat: Conditional block redesign block redesign #636

### LINKED ISSUE #636: :art:  :arrows_counterclockwise: Conditional Block Redesign: block redesign
## User stories

As a user, I want the "Conditional Block" component in the app to be redesigned to enhance its usability and visual appeal.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Technical Description
The current design is: 
image

It should be changed to:
image

The files can be found at `libs/features/convs-mgr/stories/blocks/library/conditional-block/`. 

The designs are [here](https://www.figma.com/file/ZGHzSstXOYPMFU1EiqjEHD/Untitled?type=design&node-id=1%3A4&mode=design&t=Hf7DxaUUMQ3BWy57-1)

### Acceptance Criteria

1. The block should look exactly like the designs.
2. Use of `rem`, `em` as opposed to `px`.
3. All changes should be isolated to the following  `libs/features/convs-mgr/stories/blocks/library/conditional-block/`.

### PR DESCRIPTION
**Pull request by thadeusdev**
# Description

- changed font awesome conditional icon and used the svg conditional icon.
- changed the angulat material select drop down with the vanila HTML select and chevron-down svg icon. styled background and border-radius.
- swapped the position of square custom select and label and styled them so that it displays check mark when clicked.
- styled "type variable" input and "Add Option"button  

Fixes #636

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
Screenshot from 2023-11-02 22-02-12

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/conditional-block.jsplumb.ts [hunk 1]
```diff
@@ -20,12 +20,11 @@ export function _ConditionalBlockDecoratePlumb(block: ConditionalBlock, comp: Co
...
     // Whether the anchor is target (Other Block -> This Block)
     target: true,
-    cssClass:"block_endpoint",
+    cssClass: 'block_endpoint',
 
     endpoint: 'Dot',
-    anchor: "Left",
-    maxConnections: -1
-
+    anchor: [0, 0.13, 0, 1],
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/conditional-block.jsplumb.ts:26]: Leave this as left or right for now

File: libs/features/convs-mgr/stories/blocks/library/conditional-block/src/lib/component/conditional-block/conditional-block.component.html [hunk 2]
```diff
@@ -1,44 +1,94 @@
...
+          <option selected disabled class="assess-label">
+            {{
+              'PAGE-CONTENT.BLOCK.LABELS.CONDITIONAL-BLOCK.SELECT-VARIABLE'
+                | transloco
+            }}
+          </option>
+          <option
+          *ngFor="let variable of vars$ | async"
+          [value]="variable"
+          class="assess-label"
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/conditional-block/src/lib/component/conditional-block/conditional-block.component.html:19]: Avoid indenting, when saving your files (codebase-specific rule).

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Sync your changes and resolve conflicts

File: libs/features/convs-mgr/stories/blocks/library/conditional-block/src/lib/component/conditional-block/conditional-block.component.html (hunk 2 above)
Comment (thadeusdev) [reply libs/features/convs-mgr/stories/blocks/library/conditional-block/src/lib/component/conditional-block/conditional-block.component.html:19]: Noted. i have made the requested changes. Thank you so much

**Comment by LemmyMwaura** [comment]
Closing this as it's been resolved.

**Pull request closed without merging**